applications/source/file-source/README.adoc

```diff
@@ -36,6 +36,43 @@ $$directory$$:: $$The directory to poll for new files.$$ *($$File$$, default: `$
 $$filename-pattern$$:: $$A simple ant pattern to match files.$$ *($$String$$, default: `$$<none>$$`)*
 $$filename-regex$$:: $$A regex pattern to match files.$$ *($$Pattern$$, default: `$$<none>$$`)*
 $$prevent-duplicates$$:: $$Set to true to include an AcceptOnceFileListFilter which prevents duplicates.$$ *($$Boolean$$, default: `$$true$$`)*
+
+=== metadata.store.dynamo-db
```

Comment:
This was updated as part of running the build.

Reply:
I guess this is great and exactly what we would expect from `flie-source` in regards to its configuration properties.

And if this is really a result of just generation, then it really looks good with its dividing to paragraphs according nested properties classes 😄